README.md

```diff
@@ -8,3 +8,88 @@
+
+## Features
+
+> _The codebase is currently under active development._
+
+- Framework to allow MONAI-style inference pipelines for streaming data.
+- Allows for MONAI chained transformations to be used on streaming data.
+- Inference models can be used natively in MONAI or deployed via [Triton Inference Server](https://github.com/triton-inference-server/server).
+
+## Getting Started: `x86` Development Container Setup
+
+### Creating a Local Development Container
+
+To build a developer container for your workstation simply clone the repo and run the setup script as follows.
+
+```bash
+# clone the latest release from the repo
+git clone -b main https://github.com/Project-MONAI/MONAIStream
+
+# start development setup script
+cd MONAIStream
+./start_devel.sh
+```
+
+With the successful completion of the setup script, a container will be running containing all the necessary libraries
+for the developer to start designing MONAI Stream SDK inference pipelines. The development however is limited to within
+the container and the mounted volumes. The developer may modify ``Dockerfile.devel`` and ``start_devel.sh`` to suit their
+needs.
+
+### Connecting VSCode to the Development Container
+
+To start developing within the newly created MONAI Stream SDK development container users may choose to use their favorite
+editor or IDE. Here, we show how one could setup VSCode on their local machine to start developing MONAI Stream inference
+pipelines.
+
+  1. Install [VSCode](https://code.visualstudio.com/download) on your Linux development workstation.
+  2. Install the [Remote Development Extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) and restart VSCode.
+  3. In VSCode select the icon ![VSCodeRDE](https://raw.githubusercontent.com/Project-MONAI/MONAIStream/main/docs/images/vscode_remote_development_ext.png) of the newly installed Remote Development extension on the left.
+  4. Select "Containers" under "Remote Explorer" at the top of the dialog.
+     ![VSCodeRemoteExplorer](https://raw.githubusercontent.com/Project-MONAI/MONAIStream/main/docs/images/vscode_remote_explorer.png)
+  5. Attach to the MONAI Stream SDK container by clicking the "Attach to Container" icon ![VSCodeAttachContainer](https://raw.githubusercontent.com/Project-MONAI/MONAIStream/main/docs/images/vscode_attach_container.png) on the container name.
+
+The above steps should allow the user to develop inside the MONAI Stream container using VSCode.
+
+### Run the Ultrasound Inference Sample App
+
+MONAI Stream SDK comes with example inference pipelines. Here, we run a sample app
+to perform instrument segmentation in an ultrasound video.
+
+Inside the development container perform the following steps.
+
+  1. Download the ultrasound data and models in the container.
+
+    mkdir -p /app/data
+    cd /app/data
+    wget https://github.com/Project-MONAI/monai-stream-experimental/releases/download/data/US.zip
```

Comment:
Can you just curl this file down? I thought it would get the entire archive

Reply:
Haven't tried since it went public...

Reply:
This works. resolving

Reply:
Yes, `wget` is working. Should I use `curl`?